rakam-spi/src/main/java/org/rakam/analysis/metadata/Metastore.java

```diff
@@ -26,6 +24,8 @@
 
     Map<String, Stats> getStats(Collection<String> projects);
 
+    List<String> getAttributes(String project, String collection, SchemaField field, Optional<LocalDate> startDate, Optional<LocalDate> endDate, Optional<String> query);
```

Comment:
Since the field names are unique we should use `String` instead of `SchemaField` here. Also we usually use the term `attribute` rather than `field`

Reply:
Will comply with the project

Reply:
Done